docs/book/src/topics/bring-your-own-aws-infrastructure.md

```diff
@@ -175,6 +175,19 @@ spec:
 > 
 >An incorrectly configured Classic ELB can easily lead to a non-functional cluster. We strongly recommend you let Cluster API create the Classic ELB.
 
+### Control Plane ingress rules
+
+It's possible to specify custom ingress rules for the control plane itself. To do so, add this to the AWSCluster specification:
```

Comment:
Can you clarify/define what is "the control plane itself" in this context?

Reply:
There are couple of places where we can customize ingress rules. One of them is the control plane load balancer, which is responsible for external traffic, these 2 PRs are introducing the possibility to set custom rules for it https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/4228, https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/4304. Current PR makes it possible to customize ingress rules on the control plane machine directly, in our case we want worker machines to talk with the control plane through a specific port, it's a requirement for the [RKE2](https://docs.rke2.io/install/requirements?_highlight=9345#networking).

Reply:
makes sense thanks! so this would be control plane machines/ec2 instances. Just to make sure I'm clear this could also be satisfied by setting your AdditionalSecurityGroups in those machines, correct? (Not saying that's the way to go, just making sure we put all context together)

Reply:
`AdditionalSecurityGroups` is not the best option here because we want to leverage CAPA ability to manage the underlying infrastructure. `AdditionalSecurityGroups` requires SGs to be pre-created.

Reply:
Ideally, we need a field in AWS machine template for defining security groups but I don't know how this can be combined with already existing  `AdditionalSecurityGroups ` cc @richardcase

Reply:
Indeed `AdditionalSecurityGroups []string` in `AWSCluster` is not flexible enough, while other code locations have `AdditionalSecurityGroups []AWSResourceReference` which I believe would allow searching for CAPA-managed security groups. I'll take this into account for networking code improvements.